PWGHF/Utils/utilsAnalysis.h

```diff
@@ -108,6 +111,34 @@ bool isSelectedTrackTpcQuality(T const& track, const int tpcNClustersFoundMin, c
   return true;
 }
 
+struct HfTriggerCuts : o2::framework::ConfigurableGroup {
+  std::string prefix = "hfMassCut"; // JSON group name
+
+  static constexpr float defaultDeltaMassPars3Prong[1][2] = {{-0.0025f, 0.0001f}};
+  static constexpr float defaultSigmaPars3Prong[1][2] = {{0.00796f, 0.00176f}};
+  o2::framework::Configurable<float> nSigma{"nSigma", 2.5, "Number of Sigmas for pT differential mass cut"};
+  o2::framework::Configurable<float> pTMaxDeltaMass{"pTMaxDeltaMass", 10., "Max pT to apply delta mass shift to PDG mass value"};
+  o2::framework::Configurable<float> pTMaxMassCut{"pTMaxMassCut", 8., "Max pT to apply pT differential cut"};
+  o2::framework::Configurable<o2::framework::LabeledArray<float>> deltaMassPars3Prong{"deltaMassPars3Prong", {defaultDeltaMassPars3Prong[0], 2, {"constant", "linear"}}, "delta mass parameters for HF 3 prong trigger mass cut"};
+  o2::framework::Configurable<o2::framework::LabeledArray<float>> sigmaPars3Prong{"sigmaPars3Prong", {defaultSigmaPars3Prong[0], 2, {"constant", "linear"}}, "sigma parameters for HF 3 prong trigger mass cut"};
+
+  /// Mass selection of D+ candidates to build B0 candidates
+  /// \param pTrackSameChargeFirst is the first same-charge track momentum
+  /// \param pTrackSameChargeFirst is the second same-charge track momentum
+  /// \param pTrackSameChargeFirst is the opposite charge track momentum
+  /// \param ptD is the pt of the D+ meson candidate
+  /// \return true if D+ passes selection
+  bool isSelectedDplusInMassRange(const float& invMassDplus, const float& ptD)
```

Comment:
What selection does `Selected` represent here? If you only refer to the mass range, it is already expressed by `InMassRange`.